monitoring/mongodb/dashboard.py

```diff
@@ -0,0 +1,1353 @@
+from grafanalib.core import (
+    ConstantInput,
+    DataSourceInput,
+    GridPos,
+    Heatmap,
+    HeatmapColor,
+    Repeat,
+    RowPanel,
+    Template,
+    Templating,
+    Threshold,
+    YAxis,
+)
+from grafanalib import core
+from grafanalib import formatunits as UNITS
+from scalgrafanalib import (
+    layout,
+    metrics,
+    Dashboard,
+    GaugePanel,
+    PieChart,
+    Stat,
+    Target,
+    TimeSeries,
+    Tooltip,
+    StateTimeline,
+)
+
+# Constants
+DATASOURCE = "${DS_PROMETHEUS}"
+NAMESPACE = "zenko"
```

Comment:
Need to use the template variable:

```suggestion
NAMESPACE = "${namespace}"
```

(we may also inline this, not sure the variable helps readability)